### PR TITLE
doc: remove duplicate python packages

### PR DESCRIPTION
**Pull request by hydro-b**
Small documentation fixes based on my experience of following the manual installation instructions.

### CODE REVIEW

**Comment by tchaikov** [comment]
@hydro-b could you squash these three commits into a single one? 

and, could you prefix the title of your commit message with the subcomponent your are changing ? see https://github.com/ceph/ceph/blob/master/SubmittingPatches.rst#3-describe-your-changes

also, could you add a "`Signed-off-by`" line at the end of your commit message? "git commit -s" will do the trick for you. see https://github.com/ceph/ceph/blob/master/SubmittingPatches.rst#1-sign-your-work

**Comment by hydro-b** [comment]
@tchaikov Sure, I have just done all that. Should be fine now. Thanks for the pointers.

File: doc/rbd/iscsi-target-cli-manual-install.rst [hunk 1]
```diff
@@ -147,6 +145,7 @@ on each machine that will be a iSCSI gateway:
        > cd ceph-iscsi-cli
        > python setup.py install
        > cp usr/lib/systemd/system/rbd-target-api.service /lib/systemd/system
+       > cp rbd-target-api.py /usr/bin/rbd-target-api
```
Comment (dillaman) [inline doc/rbd/iscsi-target-cli-manual-install.rst]: The ```setup.py``` script is supposed to strip the ```.py``` extension from the scripts [1][2]. Is it not working for some reason for you?

[1] https://github.com/ceph/ceph-iscsi-cli/blob/master/setup.py#L45
[2] https://github.com/ceph/ceph-iscsi-config/blob/master/setup.py#L46

Comment (hydro-b) [reply doc/rbd/iscsi-target-cli-manual-install.rst]: Hmm, it does. But it copies to "/usr/local/bin". The systemd unit file has a ExecStart path of "/usr/bin/rbd-target-api":

ExecStart=/usr/bin/rbd-target-api

I checked the status of the service: reporting it couldn't find the script. I assumed an extra "cp" step was needed, but it turns out the default install path, on Ubuntu 18.04 LTS at least, is /usr/local/bin.

Installing with "python setup.py install --install-scripts=/usr/bin" results in the script being installed in /usr/bin (without the .py extension)

Maybe add the "install --install-scripts=/usr/bin" to ensure where the script will be installed?

Comment (dillaman) [reply doc/rbd/iscsi-target-cli-manual-install.rst]: Sure, thx

**Pull request merged**